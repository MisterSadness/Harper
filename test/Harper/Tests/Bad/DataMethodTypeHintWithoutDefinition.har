value HelloWorld = {
    data = {
      name :: String;
    }
    sayHello :: String;
  };