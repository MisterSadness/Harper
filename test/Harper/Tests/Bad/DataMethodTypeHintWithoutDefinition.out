Error: type hint for function `sayHello` has no associated definition. 
During evaluation of: 
    sayHello :: String 
    
    Located at line 5 column 5 
    
Execution terminated with an error: type error.